package.json

```diff
@@ -24,7 +24,8 @@
     "release:latest": "lerna publish from-package --dist-tag latest --yes",
     "release:next": "lerna publish from-package --canary --preid next --pre-dist-tag next --yes",
     "build:docs": "cd website && yarn install && yarn build",
-    "size-limit": "size-limit"
+    "size-limit": "size-limit",
+    "prepare": "husky install"
```

Comment:
Probably it should be documented in the `CONTRIBUTING.md`

Reply:
What you mean by that? As far as i undersand prepare should [automaticlly be called](https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts) on local install. So everething should work "out of the box" for new contributors, isn't it?

Reply:
Hm, indeed. I thought it should be called manually. Nevermind